app/views/comments/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>List of Comments</h1>
+<% @comments.each do |comment| %>
+<%= comment.text %>
+<%= link_to 'Comments', comments_path %>
+<%= link_to 'Make a Comment', new_comment_path %>
+<%= link_to 'Edit your Comment', edit_comment_path(comment) %>
+<%= link_to 'Delete Comment', comment_path(comment), method: :delete %>
```

Comment:
It looks like comments are a nested resource in `routes.rb`. Because of this, you will need to use the proper path helpers, which will include `post` somewhere in their name. Take a look at `$ rails routes`...